spec/lcm/integration/user_provisioning_end_to_end_spec.rb

```diff
@@ -1,184 +1,57 @@
+                                             user_amount: 2
+  end
 
-    @test_context = {
-      project_id: @projects.first.pid, # this doesn't really matter for the runtime
-      config: LcmConnectionHelper.environment,
-      s3_bucket: Support::S3Helper::BUCKET_NAME,
-      s3_endpoint: Support::S3Helper::S3_ENDPOINT,
-      s3_key: Support::S3Helper::USER_FILTERS_KEY,
-      data_source: 's3',
-      column_name: 'value',
-      sync_mode: 'sync_domain_client_workspaces',
-      data_product: @data_product.data_product_id,
-      label: @mufs.first[:label_id],
-      users_brick_input: {
-        s3_bucket: Support::S3Helper::BUCKET_NAME,
-        s3_endpoint: Support::S3Helper::S3_ENDPOINT,
-        s3_key: Support::S3Helper::USERS_KEY
-      }
-    }
+  after(:all) do
+    @fixtures.teardown
   end
 
   describe '1 - initial MUF setup' do
     it 'provisions MUFs' do
-      test_user_filters_brick(projects: @projects,
-                              test_context: @test_context,
-                              mufs: @mufs)
+      Support::UserProvisioningHelper.test_user_filters_brick(projects: @fixtures[:projects],
+                              test_context: @fixtures[:brick_params],
+                              mufs: @fixtures[:mufs])
     end
   end
 
   describe '2 - initial user setup' do
     it 'provisions users' do
-      test_users_brick(projects: @projects,
-                       test_context: @test_context,
-                       user_data: @user_data)
+      Support::UserProvisioningHelper.test_users_brick(projects: @fixtures[:projects],
+                       test_context: @fixtures[:brick_params],
+                       user_data: @fixtures[:user_data])
     end
   end
 
   describe '3 - subsequent user setup after modification' do
     it 'provisions users' do
-      user_data = @user_data.map do |c|
+      user_data = @fixtures[:user_data].map do |c|
         c.merge('position' => 'manager')
       end
       Support::S3Helper.upload_file(ConfigurationHelper.csv_from_hashes(user_data), Support::S3Helper::USERS_KEY)
-      test_users_brick(projects: @projects,
-                       test_context: @test_context,
+      Support::UserProvisioningHelper.test_users_brick(projects: @fixtures[:projects],
+                       test_context: @fixtures[:brick_params],
                        user_data: user_data)
     end
   end
 
   describe '4 - subsequent MUF setup after modification' do
     it 'provisions MUFs' do
-      muf_complexity = 4
-      mufs = set_up_random_muf_data(@projects.first, @user_data, muf_complexity)
-      test_context = @test_context.merge(label: mufs.first[:label_id])
-      test_user_filters_brick(projects: @projects,
-                              test_context: test_context,
-                              mufs: mufs,
-                              muf_complexity: muf_complexity)
-    end
-  end
+      mufs = @fixtures[:clients].map do |client, project|
```

Comment:
warning: Similar code found in user_provisioning_end_to_end_spec.rb:46, user_provisioning_extended_spec.rb:33 (mass = 44) (flay)